analytics_extension.go

```diff
@@ -0,0 +1,45 @@
+package helix
+
+// ExtensionAnalytic ...
+type ExtensionAnalytic struct {
+	ExtensionID string `json:"extension_id"`
+	URL    string `json:"url"`
+}
+
+// ManyExtensionAnalytics ...
+type ManyExtensionAnalytics struct {
+	ExtensionAnalytics []ExtensionAnalytic `json:"data"`
+}
+
+// ExtensionAnalyticsResponse ...
+type ExtensionAnalyticsResponse struct {
+	ResponseCommon
+	Data ManyExtensionAnalytics
+}
+
+type extensionAnalyticsParams struct {
+	ExtensionID string `query:"extension_id"`
+}
+
+// GetExtensionAnalytics returns a URL to the downloadable CSV file
+// containing analytics data . Valid for 1 minute.
+func (c *Client) GetExtensionAnalytics(extensionID string) (*ExtensionAnalyticsResponse, error) {
```

Comment:
`ExtensionAnalyticsParams` should be the only parameter here. For example, when calling this method it should look like this:

```go
resp, err := GetExtensionAnalytics(&ExtensionAnalyticsParams{
	ExtensionID: extensionID,
	First:       10,
	Type:        "overview_v1",
})
```